must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-
```

Comment:
There are a few whitespace changes. They're not a big deal, but avoiding whitespace changes can make merges easier.